Packs/CommonScripts/Scripts/ExtractIndicatorsFromTextFile/ExtractIndicatorsFromTextFile.py

```diff
@@ -22,7 +22,7 @@
         data = data.decode('unicode_escape').encode('utf-8')
     # unicode_escape might throw UnicodeDecodeError for strings that contain \ char followed by ascii characters
     except UnicodeDecodeError:
-        data = data.encode('utf-8')
+        data = data.decode('iso-8859-1').encode('utf-8')
```

Comment:
we should keep the `data.encode('utf-8')` as well I think, unless this code covers it

Reply:
When running TPB with this line 
data.encode('utf-8') , the task Extract Indicators From File - Generic v2 fails.
When return to this line:
data = data.decode('iso-8859-1').encode('utf-8')
it works good. so, I don't think that we should keep that line ..